src/DependencyInjection/CompilerPass/CompilerPassHelper.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\EasyCodingStandard\DependencyInjection\CompilerPass;
+
+use Illuminate\Container\Container;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PhpCsFixer\Fixer\FixerInterface;
+use Symplify\EasyCodingStandard\Utils\PrivatesAccessorHelper;
+
+final class CompilerPassHelper
+{
+    /**
+     * @return string[]
+     */
+    public static function resolveCheckerClasses(Container $container): array
+    {
+        $serviceTypes = array_keys($container->getBindings());
+
+        return array_filter($serviceTypes, static function (string $serviceType): bool {
+            if (is_a($serviceType, FixerInterface::class, true)) {
+                return true;
+            }
+
+            return is_a($serviceType, Sniff::class, true);
+        });
+    }
+
+    public static function removeCheckerFromContainer(COntainer $container, string $checkerClass): void
+    {
+        // remove instance
+        $container->offsetUnset($checkerClass);
```

Comment:
I think forget() does the same thing and reads better

Reply:
I'll check it :+1:

Reply:
I'm just checking it. I've used this method, but it actually only removes resolved instance, not binding. So the service still exists, just it's last instance would be removed:

https://github.com/illuminate/container/blob/7ebfc9acfd5d5c7dda1ff5975927c6569651857f/Container.php#L1362-L1371

What I need is to remove the service from bindings too, is it's like it never existed:

I know the method name in Laravel container is not really nice, but it does the job better:

https://github.com/illuminate/container/blob/7ebfc9acfd5d5c7dda1ff5975927c6569651857f/Container.php#L1469-L1478